models/Style/index.d.ts

```diff
@@ -0,0 +1,63 @@
+import uuid from 'uuid-v4';
+import TextStyle from '../TextStyle';
+import Fill from '../Fill';
+import Border from '../Border';
+import Shadow from '../Shadow';
+
+declare class Style {
+  static MarkerType: {
+    none: 0;
+    openArrow: 1;
+    filledArrow: 2;
+    line: 3;
+    openCircle: 4;
+    filledCircle: 5;
+    openSquare: 6;
+    filledSquare: 7;
+  };
+
+  static WindingRule: {[key: string]: number}; // TODO
+
+  static BlendMode: {
+    Normal: 0;
+    Darken: 1;
+    Multiply: 2;
+    ColorBurn: 3;
+    Lighten: 4;
+    Screen: 5;
+    ColorDodge: 6;
+    Overlay: 7;
+    SoftLight: 8;
+    HardLight: 9;
+    Difference: 10;
+    Exclusion: 11;
+    Hue: 12;
+    Saturation: 13;
+    Color: 14;
+    Luminosity: 15;
+  };
+
+  static LayerStyle(args?: any): Style;
+
+  static TextStyle(args?: any): Style;
+
+  _class: 'style';
+  do_objectID: string;
+  startMarkerType: number;
```

Comment:
For these could we also add the values like yo do above? We probably don't need to do it now, but maybe add a todo to fill these out?

Reply:
Yes, as mentioned above, I don't know the meaning of the values exactly at present, I'll use more specified type definitions later (or someone else familiar with them).

Reply:
Sounds good. LGTM!